admin_guide/out_of_resource_handling.adoc

```diff
@@ -101,27 +101,32 @@ For example:
 ----
 kubeletArguments:
   eviction-hard: <1>
-  - memory.available<500Mi<2>
-  - nodefs.available<500Mi
-  - nodefs.inodesFree<100Mi
-  - imagefs.available<100Mi
-  - imagefs.inodesFree<100Mi
+  - memory.available<100Mi<2>
+  - nodefs.available<10%
+  - nodefs.inodesFree<5% 
+  - imagefs.available<15%
+  - imagefs.inodesFree<10% 
 ----
 
 <1> The type of eviction: Use this parameter for a xref:out-of-resource-hard-eviction-thresholds[hard eviction].
 <2> Eviction thresholds based on a specific eviction trigger signal.
 ====
 
+[NOTE]
+====
+The `inodesFree` values should always be a percentage. The other parameters can be percentage or a hard value.
```

Comment:
s/should/must
s/hard/numerical ?

Maybe, "You must provide percentage values for `inodesFree` parameters but can provide percentage or numerical values for the other parameters."

Would this note make more sense before the examples?